### PR TITLE
Add new ciphers

### PR DESCRIPTION
**Pull request by skyguard1**
Motivation:

The current ciphers is not comprehensive, some ciphers specified by rfc have not been added, see https://datatracker.ietf.org/doc/html/rfc8422
https://datatracker.ietf.org/doc/html/rfc8442
https://datatracker.ietf.org/doc/html/rfc8492

Modification:

Added new ciphers

Result:

Ciphers is more comprehensive

### CODE REVIEW

**Comment by hyperxpro** [comment]
Are CCM ciphers supported by BoringSSL or OpenSSL in Netty?

**Comment by skyguard1** [comment]
As far as I know, there is no CCM in the default list of openssl, but the cipher list can be modified through SSL_CTX_set_cipher_list() and CONF_modules_load_file(), the value of SSL_DEFAULT_CIPHER_LIST in include/openssl/ssl.h can be modified

**Comment by hyperxpro** [comment]
`Ciphers` contains ciphers that are enabled and available by OpenSSL for Netty. Also, CCM is not very popular so adding support for it might not pay off the effort.

**Comment by skyguard1** [comment]
Ok, i have removed it

**Comment by hyperxpro** [comment]
So the rest of these ciphers are fully supported by OpenSSL?

**Comment by skyguard1** [comment]
ok, I added the cipher supported by openssl, which can be seen at https://www.openssl.org/docs/manmaster/man1/openssl-ciphers.html

**Comment by skyguard1** [comment]
Is there anything else that needs to be modified in this pr?

**Comment by hyperxpro** [comment]
Can you call `OpenSsl.availableJavaCipherSuites()` and verify if your ciphers are fully supported by Netty?

File: handler/src/main/java/io/netty/handler/ssl/Ciphers.java [hunk 1]
```diff
@@ -752,6 +758,46 @@
...
+
+    /**
+     * TLS_DHE_PSK_WITH_ARIA_128_GCM_SHA256
+     */
+    public static final String TLS_DHE_PSK_WITH_ARIA_128_GCM_SHA256 = "TLS_DHE_PSK_WITH_ARIA_128_GCM_SHA256";
+
+    /**
+     * TLS_DHE_PSK_WITH_ARIA_256_GCM_SHA384
+     */
+    public static final String TLS_DHE_PSK_WITH_ARIA_256_GCM_SHA384 = "TLS_DHE_PSK_WITH_ARIA_256_GCM_SHA384";
```
Comment (normanmaurer) [inline handler/src/main/java/io/netty/handler/ssl/Ciphers.java:799]: I would be "surprised" if the `ARIA` variants are supported. That said I guess it is not a hard-requirement that all these ciphers are supported. As frankly speaking ciphers may be dropped at any point and once we make these part of a public API there is not much we can do about this.

**Pull request closed without merging**